requirements.txt

```diff
@@ -18,17 +18,18 @@ ensure<1.0.0
 flake8==3.8.0a2
 gardener-component-model
 github3.py<2.0.0
-google-api-core==1.17.0
-google-api-python-client==1.9.1
-google-auth-httplib2==0.0.3
-google-auth==1.16.0
-google-cloud-containeranalysis==0.3.1
-google-cloud-core==1.3.0
-google-cloud-storage==1.28.1
-google-resumable-media==0.5.1
-googleapis-common-protos==1.52.0
-grafeas==0.4.0
-grpc-google-iam-v1==0.12.3
+google-api-core>=1.0.0, <2.0.0
+google-api-python-client>=1.0.0, <2.0.0
+google-auth>=1.0.0, <2.0.0
+google-auth-httplib2<1.0.0
+google-cloud-containeranalysis>=2.0.0, <3.0.0
+google-cloud-core>=1.0.0, <2.0.0
+google-cloud-storage>=1.0.0, <2.0.0
+google-crc32c>=1.0.0, <2.0.0
+google-resumable-media>=1.0.0, <2.0.0
```

Comment:
just as a remark (not necessarily a request for changes): you may omit the patch-level and minor components of version restrictions afaik